### PR TITLE
Bump deps

### PR DESCRIPTION
**Pull request by rvem**
Problem: Our nixpkgs fork is a bit outdated now.

Solution: Update it with the rest of dependencies.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -71,7 +70,7 @@
         }) self.nixosConfigurations;
       };
 
-      pipelineFile = serokell-nix.lib.pipeline.mkPipelineFile self;
+      pipelineFile = serokell-nix.lib.pipeline.mkPipelineFile (self // { nixArgs = ["--impure"]; });
```
Comment (PhilTaken) [inline flake.nix:73]: Why do we need impure here? :thinking:

Comment (rvem) [reply flake.nix:73]: Otherwise checks fails due to impurity:(

Comment (PhilTaken) [reply flake.nix]: related: https://github.com/NixOS/nixpkgs/issues/179172

Comment (rvem) [reply flake.nix]: Alright, now it's properly pure

**Review by PhilTaken — APPROVED** [review]

**Pull request merged**